<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tıpfakazamisüretivitbot</string>
  </property>
  <property name="locale">
   <locale language="Turkish" country="Turkey"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="kactivitlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>46</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>            Kaç tivit atılacak?</string>
    </property>
   </widget>
   <widget class="QLabel" name="kacdakikalabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>116</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>    Kaç dakikada bir atılacak?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rastgelecheckbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rastgele sırayla at</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tweetlebutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>84</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Tweetle!</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="gecensurelcd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="atilantweetlcd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gecensurelabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>    Geçen süre</string>
    </property>
   </widget>
   <widget class="QLabel" name="atlantweetlabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Atılan Tweet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secimlerikaydetbutton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>141</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçimleri kaydet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kactivitline">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="kacdakikaline">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="anahtarlarmenubar">
    <property name="title">
     <string>Tweeter Anahtarları</string>
    </property>
    <addaction name="tweeteranahtarlarinigir"/>
   </widget>
   <widget class="QMenu" name="menuTweet_Mention_Hashtag">
    <property name="title">
     <string>Tweet, Mention, Hashtag</string>
    </property>
    <addaction name="actionTweet"/>
    <addaction name="actionMention"/>
    <addaction name="actionHashtag"/>
   </widget>
   <addaction name="anahtarlarmenubar"/>
   <addaction name="menuTweet_Mention_Hashtag"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="tweeteranahtarlarinigir">
   <property name="text">
    <string>Tweeter Anahtarlarını Gir</string>
   </property>
  </action>
  <action name="actionTweet">
   <property name="text">
    <string>Tweet</string>
   </property>
  </action>
  <action name="actionMention">
   <property name="text">
    <string>Mention</string>
   </property>
  </action>
  <action name="actionHashtag">
   <property name="text">
    <string>Hashtag</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
